test/evergreen/build_windows.ps1

```diff
@@ -0,0 +1,37 @@
+param (
+    [bool]$configure = $false,
+    [bool]$build = $false
+)
+
+# Source the vcvars to ensure we have access to the Visual Studio toolchain
+pushd "C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Auxiliary\Build"
```

Comment:
Not sure if there's a way in PowerShell to make the script more future-proof in terms of VS version reference, say if we decide to switch Evergreen tasks to a newer Windows distro with a newer version of VS we don't need to worry about bumping the version in the path here.

Reply:
Good point, I'll look to see if I can pass the VC toolchain directory as a parameters

Reply:
Updated! 
To make it as configurable as possible, I've updated the script to take in the path to the actual vcvars files. I noticed the path and file has changed over different Visual Studio releases. So I figured it would better if evergreen can actually specify the vcvars file to source (since this may be subject change).

Reply:
I like this change, thanks!